examples/device/data-normal.json

```diff
@@ -0,0 +1,5 @@
+{
+	"appId": "DEVICE",
+	"messageType": "STATUS",
```

Comment:
Double checking with Henrik that the messageType is "STATUS". nRF Cloud doesn't care really, so it won't break anything, but this spec should be consistent with the FW.